org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/parser/Parser.java

```diff
@@ -4092,6 +4092,10 @@ protected void consumeExitTryBlock() {
 	if(this.currentElement != null) {
 		this.restartRecovery = true;
 	}
+	//Javadoc
+	if (this.scanner.commentStarts[0] > 0 && this.scanner.commentStops[0] > 0) {
```

Comment:
I am not sure we want to do this here. It would better to be handled in the specific subclass Parser, like the CommentRecorderParser().

Reply:
Done